packages/i18n/load-i18n.js

```diff
@@ -1,24 +1,73 @@
 import { addLocaleData } from 'react-intl';
 import { getMatchingMomentCode } from './utils';
 
+const getReactIntlChunkImport = lang => {
+  let chunkImport;
+  switch (lang) {
+    case 'en':
+      chunkImport = import(/* webpackChunkName: "react-intl-data-en" */ 'react-intl/locale-data/en');
+      break;
+    case 'de':
+      chunkImport = import(/* webpackChunkName: "react-intl-data-de" */ 'react-intl/locale-data/de');
+      break;
+    case 'es':
+      chunkImport = import(/* webpackChunkName: "react-intl-data-es" */ 'react-intl/locale-data/es');
+      break;
+    default:
+      chunkImport = import(/* webpackChunkName: "react-intl-data-en" */ 'react-intl/locale-data/en');
+  }
+  return chunkImport;
+};
+
+const getMomentChunkImport = momentLocaleCode => {
+  let chunkImport;
+  switch (momentLocaleCode) {
+    case 'en':
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-en-gb" */ 'moment/locale/en-gb');
+      break;
+    case 'de':
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-de" */ 'moment/locale/de');
+      break;
+    case 'es':
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-es" */ 'moment/locale/es');
+      break;
+    default:
+      chunkImport = import(/* webpackChunkName: "i18n-moment-locale-en-gb" */ 'moment/locale/en-gb');
+  }
+  return chunkImport;
+};
+
+const getLocalizedStringsChunkImport = lang => {
+  let chunkImport;
+  switch (lang) {
+    case 'en':
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-en" */ './data/en.json');
+      break;
+    case 'de':
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-de" */ './data/de.json');
+      break;
+    case 'es':
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-es" */ './data/es.json');
+      break;
+    default:
+      chunkImport = import(/* webpackChunkName: "react-intl-localized-strings-en" */ './data/en.json');
+  }
+  return chunkImport;
+};
```

Comment:
This is totally fine. It‘s one wall of code. Things to change are easily spotted snd should not change every other week after all.

Reply:
Yeah... it's not the prettiest. Once all of the language related changes have been made we will need to update the readme docs on how to add new languages. But like you said, it's not happening everyday 🤷‍♂️. Updating a few files is small work compared to translating thousands of strings 😆